client/my-sites/site-settings/jetpack-credentials/credentials-configured/index.jsx

```diff
@@ -40,15 +49,48 @@ class CredentialsConfigured extends Component {
 			formIsSubmitting,
 			siteId,
 			updateCredentials,
+			deleteCredentials,
 			translate,
 		} = this.props;
 
+		const isRevoking = this.state.isRevoking;
 		const protocol = get( this.props.mainCredentials, 'protocol', 'SSH' ).toUpperCase();
 		const protocolDescription = this.getProtocolDescription( protocol );
 
-		if ( isPressable ) {
+		if ( isRevoking ) {
 			return (
 				<CompactCard className="credentials-configured">
+					<p>
+						{ translate(
+							"Your site's server was automatically connected to Jetpack to " +
+								'perform backups, rewinds, and security scans. You do not have to ' +
+								'configure anything further, but you may revoke the credentials if necessary.'
+						) }
+					</p>
+					<div className="credentials-configured__revoke-actions">
+						<Button
+							className="credentials-configured__revoke-button"
+							borderless={ true }
+							onClick={ this.handleRevoke }
+						>
+							<Gridicon
+								className="credentials-configured__revoke-icon"
+								icon="link-break"
+								size={ 18 }
+							/>
+							{ translate( 'Revoke credentials' ) }
+						</Button>
+						<Button primary onClick={ this.toggleRevoking }>
+							{ translate( 'Stay connected' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Stay Connected' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).